roles/keystone-defaults/defaults/main.yml

```diff
@@ -60,18 +60,25 @@ keystone:
     sp:
       # Enable keystone to be a service provider
       # for keystone to keystone federation
-      k2k:
-        enabled: False
+      saml:
```

Comment:
Changing this format means some environments will have to have their configuration modified. And ursula_env will have to be modified.